src/main/java/org/synyx/urlaubsverwaltung/calendar/ICalService.java

```diff
@@ -130,6 +132,10 @@ private Calendar prepareCalendar(List<Absence> absences, ICalType method) {
         event.getProperties().add(new Uid(generateUid(absence)));
         event.getProperties().add(generateAttendee(absence));
 
+        if (absence.isHolidayReplacement()) {
```

Comment:
It depends :-)
Currently, we do only model `DEFAULT` and `HOLIDAY_REPLACEMENT` as `AbsenceTypes`. If we turn it around, we will not get the ical default behavior if we would add another type

Reply:
hmm....took a look at the code and we do not know that. I thought about it, because maybe it would be nice if the office receives a ical of another persons absence and it will also marked as `TRANSPARENT`

Reply:
But let's go with this at first and then look further.